docs/en/pact-reference.md

```diff
@@ -388,31 +388,44 @@ as a unit.
 admin authorization schemes for modules and tables. Definition creates the keysets
 in the runtime environment and stores their definition in the global keyset database.
 
+#### Namespace declaration {#namespacedefinition}
+
+[Namespace](#namespaces) declarations provide a unique prefix for keysets and modules defined within the namespace scope. All smart contract definitions must occur within a namespaced scope, which is initiated by defining a namespace using [define-namespace](pact-functions.html#define-namespace), and entering the namespace scope by issuing the [namespace](pact-functions.html#namespace) command. The global (i.e. empty) namespace is reserved for Kadena code.
```

Comment:
```suggestion
[Namespace](#namespaces) declarations provide a unique prefix for modules defined within the namespace scope. All smart contract definitions must occur within a namespaced scope, which is initiated by defining a namespace using [define-namespace](#define-namespace), and entering the namespace scope by issuing the [namespace](#namespace) command. The global (i.e. empty) namespace is reserved for Kadena code.
```

Reply:
Also get rid of "The global ..." and make a para on Chainweb namespaces vs private blockchain (where namespaces are fine).

Chainweb is a WIP.